src/main/index.ts

```diff
@@ -75,6 +77,8 @@ async function main() {
 
   // create window manager and open app
   windowManager = new WindowManager(proxyPort);
+
+  extensionLoader.extensions.replace(await extensionManager.load())
```

Comment:
So, currently all extensions only `renderer` specific? Or how to customize, for example, app global menu from extension? (probably out of scope / needs to create separate issue).

Reply:
I think we can support `main` extensions just by adding a call to `extensionLoader.autoEnableOnLoad()` before this line. Probably should be done in a separate PR.

Reply:
I'll create issue about this. Played a bit around it today, and it's not that simple.